src/cleanfigure.m

```diff
@@ -206,8 +203,9 @@ function pruneOutsideBox(meta, handle)
   %hasMarkers = ~strcmp(marker,'none');
   hasLines = true;
```

Comment:
I opened an Issue for that. I would say, that this is unrelated to the restructure here, and i would prefer to get it landed rather soon, as it is a prerequisite for 3D handling.